### PR TITLE
chore(cvsb-19465): update README and scripts

### PR DESCRIPTION
**Pull request by JonLazariniDD**
## Ticket title

One line description
[link to ticket number]()

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,169 @@
...
+Data is used is made available to VTA for searching a vehicle.
 
-https://github.com/UKHomeOffice/repo-security-scanner
+### End to end design
 
-- After installing, run with `git log -p | scanrepo`.
+[All in one view](https://wiki.dvsacloud.uk/pages/viewpage.action?pageId=79254695)
 
-These will be run as part of prepush so please make sure you set up the git hook above so you don't accidentally introduce any new security vulnerabilities.
+### Defect microservice
```
Comment (bristol-mike-jones) [inline README.md:37]: Typo - This is Preparers

**Review by bristol-mike-jones — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (JonLazariniDD) [reply README.md]: Thanks for spotting that! fixed, feel free to resolve if happy.

**Review by Daniel-Searle — APPROVED** [review]

**Review by githanium — APPROVED** [review]
LGTM

**Review by sriniarumugam — APPROVED** [review]
LGTM

**Pull request merged**